insights/specs/datasources/luks_devices.py

```diff
@@ -123,3 +124,28 @@ def luks_data_sources(broker):
         return datasources
 
     raise SkipComponent
+
+
+@datasource(LuksDump, HostContext)
```

Comment:
Since the `LuksDump` is required for this datasource, please add it to the `insights/collect.py` too.

Reply:
@xiangce @danzatt why not just use parsers and a collector instead of adding a datasource that has to run on a customer systems?